docs/quickstart_guide.md

```diff
@@ -161,7 +162,8 @@ SSD-equipped nodes are highly recommended. Full list of CloudLab nodes can be fo
 5. Start `vHive` in a background terminal named `vhive`:
     ```bash
     # EITHER
-    sudo screen -dmS vhive bash -c "./vhive > >(tee -a /tmp/vhive-logs/vhive.stdout) 2> >(tee -a /tmp/vhive-logs/vhive.stderr >&2)"
```

Comment:
Again this will be confusing, also I'm not sure if this command should be deleted. I think it is there to run vhive in various modes with the firecracker

Reply:
These are my tweaks that got into the diff unintentionally, dropped them.